recipes-openxt/xenclient/uid_git.bb

```diff
@@ -18,7 +13,7 @@ SRC_URI += "file://uid_dbus.conf \
 
 S = "${WORKDIR}/git"
 
-inherit xenclient update-rc.d
```

Comment:
uid doesn't seem to "inhibit rm-work" anymore, probably because it doesn't depend on xenclient.
It doesn't matter for the builder, but it probably does for development builds...
There might be other recipes that have this problem, please have a look.

Reply:
I removed instances where recipes inherit `xenclient.bbclass` because it appeared that [`rm_work`](http://www.yoctoproject.org/docs/current/ref-manual/ref-manual.html#ref-classes-rm-work) is not done by default. It would need to inherit `rm_work.bbclass` then use exclusion through `RM_WORK_EXCLUDE` or disable `do_rm_work` like `xenclient.bbclass` does.

Does the builder have `INHERIT += "rm_work"` in the `local.conf` file?

Reply:
I checked build that don't include this PR, and the uid work directory also gets removed there, so my comment is irrelevant.
Will merge once https://github.com/OpenXT/meta-openxt-haskell-platform/pull/8 is settled.